components/Dropdown/Dropdown.js

```diff
@@ -2,24 +2,23 @@ class Dropdown {
   constructor(element) {
     
     // Assign this.element to the dropdown element
-    this.element;
+    this.element = element;
     
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
+    this.button = this.element.querySelector('.dropdown-button');
     
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
+    this.content = this.element.querySelector('.dropdown-content');
     
     // Add a click handler to the button reference and call the toggleContent method.
-    this.button.addEventListener('click', () => {
-
-    })
+    this.button.addEventListener('click', this.toggleContent.bind(this))
```

Comment:
Great job, If you use the ( ) => syntax, it will remove the need to use the bind() keyword to bind toggleContent to (this)